docs/apis-clients/operate-api/index.md

```diff
@@ -22,29 +22,24 @@ For example: [http://localhost:8080/swagger-ui.html](http://localhost:8080/swagg
 
 You need authentication to access the API endpoints.
 
-### Authentication in the cloud
+### Authentication for C8 SaaS (Cloud) usage
```

Comment:
❌ No C8, please - spell out Camunda Platform 8.

Alternatively, "Authentication for SaaS" would be sufficient. Website uses the acronym, so it feels safe for us to use too: Platform > Sign up for SaaS.

Reply:
Resolved in latest commit 👍